### PR TITLE
added Toast and  navbar enhance

### PR DESCRIPTION
**Pull request by anjali1102**
Check HERE   👉  [Preview](https://rapidui-fb6e00118-anjali1102.vercel.app/)

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,20 +1,38 @@
...
 const togglerNew = document.querySelector(".toggler-new");
 
-console.log(togglerNew);
-
-togglerNew.addEventListener("click", () => {
-  console.log("hello");
-  const nav = document.querySelector(".component-list");
-  nav.classList.toggle("show");
-});
+console.log(togglerNew);
```
Comment (SK02K1) [inline index.js]: we can remove this before merging
```suggestion
```

File: index.js [hunk 2]
```diff
@@ -1,20 +1,38 @@
...
 };
 
+togglerNew.addEventListener("click", () => {
+  console.log("hello");
+  const nav = document.querySelector(".component-list");
+  nav.classList.toggle("show");
+});
 const toggler = document.querySelector(".toggler");
 toggler.addEventListener("click", toggleNav);
 console.log(toggler);
```
Comment (SK02K1) [inline index.js]: we can remove this before merging
```suggestion
```

File: index.js [hunk 3]
```diff
@@ -1,20 +1,38 @@
...
+  handleSucessToast("This is success toast.")
+);
 
 const toggleNav = () => {
   const nav = document.querySelector(".component-list");
   nav.classList.toggle("show");
 };
 
+togglerNew.addEventListener("click", () => {
+  console.log("hello");
```
Comment (SK02K1) [inline index.js]: we can remove this before merging
```suggestion
```

File: docs.css [hunk 4]
```diff
@@ -364,3 +375,34 @@ header button {
...
+
+.nav-links li {
+  padding-right: 1rem;
+  font-weight: bold;
+  font-size: 1rem;
+}
+
+.social-links {
+  display: flex;
+  /* color: var(--white); */
```
Comment (SK02K1) [inline docs.css]: we can remove this before merging
```suggestion
```

File: docs.css [hunk 5]
```diff
@@ -119,17 +119,33 @@ li {
...
+  /* overflow: hidden; */
 }
 
 .navbar-logo {
-  color: var(--white);
+  color: var(--black);
   font-size: 1.4rem;
   text-decoration: none;
-  width: 100%;
+  /* width: 100%; */
```
Comment (SK02K1) [inline docs.css]: we can remove this before merging
```suggestion
```

File: docs.css [hunk 6]
```diff
@@ -119,17 +119,33 @@ li {
...
   align-items: center;
-  padding: 1em;
   justify-content: space-between;
+  height: 85px;
+  padding: 0 2rem;
+  background-color: var(--black);
   flex-wrap: wrap;
-  background-color: #1877f2;
+
+  /* overflow: hidden; */
```
Comment (SK02K1) [inline docs.css]: we can remove this before merging
```suggestion
```

**Review by SK02K1 — APPROVED** [review]
Just remove those commented code rest looks good

**Review by SK02K1 — APPROVED** [review]

File: docs.css [hunk 7]
```diff
@@ -238,8 +239,19 @@ header button {
...
 .component-list-item:hover {
-  background-color: #1877f2;
+  background-color: black;
+  color: white;
+  transition: 0.5s ease;
+}
+
+.component-list-item:active {
+  background-color: black;
   color: white;
```
Comment (jeetsdev) [inline docs.css:255]: Here you can use color variables. As I can see you are using it consistently, seems like you missed it here.

**Review by jeetsdev — APPROVED** [review]
Everything looks good.

**Review by aniketxparihar — APPROVED** [review]

**Review by aniketxparihar — COMMENTED** [review]
Looks good. It's a good practice to use variables if repeating the same color many times so try to use that

**Pull request merged**